images/win/scripts/Installers/Validate-NodeLts.ps1

```diff
@@ -26,6 +26,16 @@ else
     exit 1
 }
 
+if((Get-Command -Name 'lerna'))
```

Comment:
I think we don't need double brackets in this condition, `if(Get-Command -Name 'lerna')` should be enough.